src/main/java/com/redhat/devtools/lsp4ij/features/documentSymbol/LSPDocumentSymbolSupport.java

```diff
@@ -73,9 +73,11 @@ protected CompletableFuture<List<DocumentSymbolData>> doLoad(DocumentSymbolParam
     }
 
     private static CompletableFuture<List<DocumentSymbolData>> getDocumentSymbolsFor(@NotNull DocumentSymbolParams params,
+                                                                                     @NotNull PsiFile file,
```

Comment:
Is there a reason that this signature changed to reverse the order of the second and third parameters? Perhaps this order just makes more sense, but wanted to confirm this wasn't just an accident with no motivation.

Reply:
I did that to align this rule with other LSP*Support (see LSPCompletionSupport).